### PR TITLE
Amplification uncertainty

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

**Comment by jpa198** [comment]
Not sure why CB_amp_old was there, but I deleted it as it still exists in git

File: qcore/siteamp_models.py [hunk 1]
```diff
@@ -28,9 +28,36 @@
...
+    :param amplification_factors: numpy array of amplification factors
+    :param frequencies: numpy array of frequencies for the amplification factors
+    :param seed: Seed to use for the prng. None for a random seed
+    :return: amplification factors with uncertainty applied
+    """
+    sigma_x = (
+        0.6167
+        - 0.1495 / (1 + np.exp(-3.6985 * np.log(frequencies / 0.7248)))
+        + 0.3640 / (1 + np.exp(-2.2497 * np.log(frequencies / 13.457)))
+    )
```
Comment (joelridden) [inline qcore/siteamp_models.py:50]: Is there a paper that could be linked for these numbers? That could be added to the docstring maybe

Comment (jpa198) [reply qcore/siteamp_models.py:50]: No, these magic numbers are from @sarahjaneneill. She may have a paper to cite though.

**Comment by jpa198** [comment]
Sarah has also approved these changes

File: qcore/siteamp_models.py (hunk 1 above)
Comment (jpa198) [reply qcore/siteamp_models.py:50]: And here's the magic: https://pubs.geoscienceworld.org/ssa/bssa/article/107/6/2774/519322/Interfrequency-Correlations-among-Fourier-Spectral

**Review by joelridden — APPROVED** [review]

**Review by sungeunbae — APPROVED** [review]

**Pull request merged**